pkgs/development/compilers/gcc/common/configure-flags.nix

```diff
@@ -127,11 +127,18 @@ let
           }"
           "--enable-nls"
         ]
-        ++ lib.optionals (targetPlatform.libc == "uclibc" || targetPlatform.libc == "musl") [
-          # libsanitizer requires netrom/netrom.h which is not
-          # available in uclibc.
-          "--disable-libsanitizer"
-        ]
+        ++
+          lib.optionals
+            (lib.elem targetPlatform.libc [
+              "mlibc"
+              "musl"
+              "uclibc"
+            ])
+            [
+              # libsanitizer requires netrom/netrom.h which is not
+              # available in uclibc.
+              "--disable-libsanitizer"
+            ]
```

Comment:
Is this a glibc-specific header? Would it be better to disable it on all non-GNU platforms?

Reply:
Could we get that information from upstream included in a comment then?